todo_app/templates/index.html

```diff
@@ -11,8 +11,15 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for my_item in todo %}
+       <li  >{{my_item.title}}</li>
+        {% endfor %}
+
       </ul>
     </div>
   </div>
+  <form action= "/newitem" method="post" > 
+    <input type="text" name = "title">
+    <input type="submit" value= "Add your new item in the box">
```

Comment:
You could add a `required` attribute to this tag to prevent people submitting blank to-dos.